Gemfile

```diff
@@ -20,7 +20,7 @@ end
 
 gem 'acts_as_tenant'
 gem 'jbuilder', '~> 2.0'
-gem "pg", "~>0.18.2", :require => false
+gem "pg", "~> 1.0", :require => false
```

Comment:
@gmcculloug, @mkanoor - This change is fine assuming for codegen?  I know the racecar1 change is fine.